mods/ra/maps/evacuation/evacuation.lua

```diff
@@ -0,0 +1,365 @@
+DeathThreshold =
+{
+	Easy = 200,
+	Normal = 100,
+}
+
+TanyaType = "e7"
+TanyaStance = "AttackAnything"
+if Map.Difficulty ~= "Easy" then
+	TanyaType = "e7.noautotarget"
+	TanyaStance = "HoldFire"
+end
+
+RepairTriggerThreshold =
+{
+	Easy = 50,
+	Normal = 75
+}
+
+Sams = { Sam1, Sam2, Sam3, Sam4 }
+TownUnits =
+{
+	Einstein, Engineer,
+	TownUnit01, TownUnit02, TownUnit03, TownUnit04, TownUnit05, TownUnit06, TownUnit07,
+	TownUnit08, TownUnit09, TownUnit10, TownUnit11, TownUnit12, TownUnit13, TownUnit14
+}
+
+ParabombDelay = DateTime.Seconds(30) 
+ParatroopersDelay = DateTime.Minutes(5)
+Paratroopers =
+{
+	{
+		-- `proxy` cannot be assigned in global scope since a player is needed to create the actor
```

Comment:
Ah wait, we might want to destroy the proxy actor as we don't use it again anyway, so I'd suggest following instead:
```
Utils.Do(Paratroopers, function(para)
    local proxy = Actor.Create(para.proxy, false, { Owner = soviets })
    local units = proxy.SendParatroopersFrom(para.entry, para.drop)
    proxy.Destroy()
    Utils.Do(units, function(unit)
[...]